.gitignore

```diff
@@ -1,3 +1,3 @@
-/target
+**/target
```

Comment:
I don't understand this change.  The tough repo has a Cargo.toml declaring the subdirectories to be in a workspace, and when you build (even from a subdirectory) the results go into `/target` in the root of the repo, not a subdirectory.  What are you doing when you see `target` somewhere else?

Reply:
I ran `cargo run` inside `tuftool/`, and that generated `tuftool/target`:

```bash
Untracked files:
  (use "git add <file>..." to include in what will be committed)
        tough/target/
        tuftool/target/
```

Reply:
I don't have that behavior - `build` and `run` always put artifacts in the top-level `target`.  Let's talk offline about our setups.

Reply:
I re-ran `cargo run` inside `tuftool`, I didn't see the folder. I'll remove the commit that updates the `.gitignore` file.